src/Documents/Session/Advanced.ts

```diff
@@ -0,0 +1,11 @@
+import {DocumentSession} from './DocumentSession';
+
+export class Advanced {
```

Comment:
Please rename to AdvancedSessionOperations to match C# client. https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Client/Documents/Session/IAdvancedSessionOperations.cs#L15